08week/jquery-counter/script.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+console.log("I am Matt");
+/*
+$('div').click(function(){
+    var current = $('#counter').text();
+    console.log ("current counter is " + current);
+    var currentAsNumber = parseInt(current);
+    var newCounter = currentAsNumber +1;
+    $('#counter').text(newCounter);
+    
+})
+*/
+/*
+$("#counter-1").click(function(){
+    var clickOne = 1;
+    for (var i = 0; i < 0)
+    
+    console.log();  
+})
+*/
+/*
+$(document).ready(function() {
+  var num = 0;
+  $('#counter-1').click(function() {
+    num++;
+    $(this).text(num);
+    
+  })
+});
+
+$(document).ready(function() {
+  var num = 0;
+  $('#counter-10').click(function() {
+    num = num + 10;
+    $(this).text(num);
+  })
+});
+
+$(document).ready(function() {
+  var num = 0;
+  $('#counter-100').click(function() {
+    num = num + 100;
+    $(this).text(num);
+      
+  })
+});
+*/
+
+$('.incrementer').click(function(){
```

Comment:
Most of this logic is correct, but it just looks like you did not take the time to finish it off?